modules/catalog/src/main/java/org/apache/ignite/internal/catalog/CatalogManagerImpl.java

```diff
@@ -648,16 +649,21 @@ private SystemView<?> createSystemViewColumnsView() {
     }
 
     private SystemView<?> createZonesView() {
-        return SystemViews.<CatalogZoneDescriptor>clusterViewBuilder()
+        return SystemViews.<ZoneWithDefaultMarker>clusterViewBuilder()
                 .name("ZONES")
-                .addColumn("NAME", STRING, CatalogZoneDescriptor::name)
-                .addColumn("PARTITIONS", INT32, CatalogZoneDescriptor::partitions)
-                .addColumn("REPLICAS", INT32, CatalogZoneDescriptor::replicas)
-                .addColumn("DATA_NODES_AUTO_ADJUST_SCALE_UP", INT32, CatalogZoneDescriptor::dataNodesAutoAdjustScaleUp)
-                .addColumn("DATA_NODES_AUTO_ADJUST_SCALE_DOWN", INT32, CatalogZoneDescriptor::dataNodesAutoAdjustScaleDown)
-                .addColumn("DATA_NODES_FILTER", STRING, CatalogZoneDescriptor::filter)
-                .addColumn("IS_DEFAULT_ZONE", BOOLEAN, isDefaultZone())
-                .dataProvider(SubscriptionUtils.fromIterable(() -> catalogAt(clockService.nowLong()).zones().iterator()))
+                .addColumn("NAME", STRING, wrapper -> wrapper.zone.name())
+                .addColumn("PARTITIONS", INT32, wrapper -> wrapper.zone.partitions())
+                .addColumn("REPLICAS", INT32, wrapper -> wrapper.zone.replicas())
+                .addColumn("DATA_NODES_AUTO_ADJUST_SCALE_UP", INT32, wrapper -> wrapper.zone.dataNodesAutoAdjustScaleUp())
+                .addColumn("DATA_NODES_AUTO_ADJUST_SCALE_DOWN", INT32, wrapper -> wrapper.zone.dataNodesAutoAdjustScaleDown())
+                .addColumn("DATA_NODES_FILTER", STRING, wrapper -> wrapper.zone.filter())
+                .addColumn("IS_DEFAULT_ZONE", BOOLEAN, wrapper -> wrapper.isDefault)
```

Comment:
it would be nice of you to specify type of columns explicitly according to System view registration guidelines. Since this is totally unrelated to original issue, feel free to ignore this comment

Reply:
Done